provisioning/miniprov/hare_mp/main.py

```diff
@@ -529,6 +529,11 @@ def main():
                    help_str='Validates configuration pre-requisites',
                    handler_fn=noop)
 
+    add_subcommand(subparser,
+                   'upgrade',
+                   help_str='Upgrades the Hare RPM (Currently unavailable)',
```

Comment:
Why "Currently unavailable"? When will it become available?

RPM is upgradable per se. At this point there is nothing that we need to do in scope of this call (and I don't think that it will change in the near future). So this message about unavailability is misleading. Let's remove it.

Reply:
Done!